components/Tabs/docs/theming-extensions.md

```diff
@@ -0,0 +1,38 @@
+### Theming Extensions (Beta)
+
+ `MDCTabBar` supports Material Theming using a Container Scheme. The code is currently
+in Beta readiness, which means you will need to follow the [instructions for adding the
+MaterialComponentsBeta podspec to your
+project](https://github.com/material-components/material-components-ios/blob/73bdc03c2bd2abd032b0b69f05cd76928361aa37/contributing/beta_components.md#beta-program-for-components).
+There are two variants for Material Theming of a MDCTabBar, which are the Primary Variant
+color and the Surface Variant.
+
+ <!--<div class="material-code-render" markdown="1">-->
+
+ #### Swift
+
+ ```swift
+// Import the TabBar Theming Extensions module
+import MaterialComponentsBeta.MaterialTabs_MaterialTheming
+ ...
+ // Apply your app's Container Scheme to the TabBar controller
+let containerScheme = MDCContainerScheme()
+ // Either Primary Theme
+tabBar.applyPrimaryTheme(withScheme: containerScheme)
+ // Or Surface Variant Theme
+tabBar.applySurfaceVariantTheme(withScheme: containerScheme)
+```
+
+ #### Objective-C
+
+ ```objc
+// Import the TabBar Theming Extensions header
+#import <MaterialComponentsBeta/MaterialTabBar+MaterialTheming.h>
+ ...
+ // [[Apply]] your app's Container Scheme to the TabBar controller
+MDCContainerScheme *containerScheme = [[MDCContainerScheme alloc] init];
+ // Either Primary Theme
+[self.tabBar applyPrimaryThemeWithScheme:containerScheme];
+ // Or Surface Theme
+[self.tabBar applySurfaceVariantThemeWithScheme:containerScheme];
+```
```

Comment:
Is this missing the ending `div`?

Reply:
Looks like this wasn't updated in the README.md. Did you forget to regenerate the documentation?

Reply:
I think I regenerated it. The reference commit: c5b9b9f